test-it/ExpectedResults/ArrayConcatenateUninitialized.vtr

```diff
@@ -0,0 +1 @@
+Result: (((1 2 3 4) (0 0 0 8) (0 0 0 12) (0 0 0 16) (0 0 0 20) (0 0 0 24) (0 0 0 28) (0 0 0 32) (0 0 0 36)) ((1 2 3 4) (0 0 0 8) (0 0 0 12) (0 0 0 16) (0 0 0 20) (0 0 0 24) (0 0 0 28) (0 0 0 32) (0 0 0 36)))
```

Comment:
@cglzaguilar Is this the same output when this VI is run Desktop target?

Reply:
Yes, it is the same output.